requirements/base.in

```diff
@@ -0,0 +1,19 @@
+-c constraints.txt
+
+attrs
+pyblake2
+django
+django-cors-headers
+django-environ
+django-extensions
```

Comment:
It does not look like we are using it? It needs to be added to `INSTALLED_APPS` to work?

https://django-extensions.readthedocs.io/en/latest/installation_instructions.html#configuration

Reply:
Removed, it seems that it is not being used.